sql/mozfun/norm/fenix_build_to_date/metadata.yaml

```diff
@@ -1,8 +1,12 @@
+---
 friendly_name: Fenix Build To Datetime
 description: |
-  Convert the Fenix client_info.app_build-format string to a DATETIME. May return NULL on failure.
+  Convert the Fenix client_info.app_build-format string to
+  a DATETIME. May return NULL on failure.
 
-  Fenix originally used an [8-digit app_build format](https://github.com/mozilla-mobile/fenix/blob/c7283447/automation/gradle/versionCode.gradle#L12-L16)
+  Fenix originally used an [8-digit app_build format]
+  (https://github.com/mozilla-mobile/fenix/blob/c7283447/automation
+  /gradle/versionCode.gradle#L12-L16)
```

Comment:
I don't think we should be breaking URLs across multiple lines. Could you enable the following setting and undo any breaks in URLs?

```
rules:
  line-length:
    max: 80
    allow-non-breakable-words: true
    allow-non-breakable-inline-mappings: true
```

Reply:
where do I find that?

Reply:
See https://yamllint.readthedocs.io/en/stable/configuration.html#configuration

```yaml
extends: default

rules:
  line-length:
    max: 80
    allow-non-breakable-words: true
    allow-non-breakable-inline-mappings: true
```

Reply:
I'm still getting the "line too long" error, even when I enable this setting somehow

Reply:
Did you modify the yamllint statement to read from the configuration file?

Reply:
No I'd been using the custom config without config file route. I'll try it the other way but first, do I add the enabled setting to the config file before that?

Reply:
Yes, add this setting and use the yamllint command pointing to the configuration file. It should allow for less line breaks then there are currently.